app/services/authn_context_resolver.rb

```diff
@@ -100,4 +119,11 @@ def acr_ial_component_values
       component_value.name.include?('ial') || component_value.name.include?('loa')
     end
   end
+
+  def biometric_is_required?(result)
+    result.
+      component_values.
+      map(&:name).
+      include?(Saml::Idp::Constants::IAL2_BIO_REQUIRED_AUTHN_CONTEXT_CLASSREF)
```

Comment:
wait can't we do this?
```suggestion
    result.biometric_comparison?
```

Reply:
and if not, maybe we need to split into `result.biometric_requested?` vs `result.biometric_required?` (could be a follow-up but my expectation is that the point of the result class is to define capabiltiies so we don't have to check for specific ACRs throughout the codebase)

Reply:
we definitely could do something like that! the initial implementation did add a `biometric_comparison_required`, but [hooper thought that was a little confusing](https://github.com/18F/identity-idp/pull/10948/files#r1688168506) as someone trying to work downstream with the code. but maybe requested vs required would be clearer? what do you think?

Reply:
the reason we need both is because we need a way to "fall back" onto our base identity proofing for partners who don't want to force their users to go through it again.

Reply:
yeah I think the fact that this PR defines `biometric_is_required?` means definitely we need a way to distinguish, so I would vote to rename the existing one, so yes there are both required + requested, but that could def be a candidate for a separate PR to keep this one focused

Reply:
okay yeah -- i can add this thought to the SAML implementation ticket (which, work on that can start as soon as this is merged in), since i think there won't be a ton of work in that, so it can focus a little more on optimizations like this.